dci.pl

```diff
@@ -53,6 +53,15 @@
     format(string(Text), "** [~w][~w] new component uploaded ~w (first one)", [Product, Topic, NewComponent]),
     notify(Text, []).
 
+dci_solver(_) :-
```

Comment:
it could more prolog like:

dci_solver(_) :-
    get_fact(dci_component(_, Topic, NewComponent)),
    get_fact(puddle_info(Topic, _, "latest", Puddle)),
    split_string(NewComponent, " ", "", [_, ComponentPuddle]),